### PR TITLE
chore(docs): add release-note-0.3

### PR DESCRIPTION
**Pull request by easy1090**
## Summary
chore(docs): add release-note-0.3

### CODE REVIEW

File: packages/document/docs/zh/blog/release/release-note-0.3.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# Rsdoctor v0.2-0.3 Release Note
+
+### 1. 自定义拓展规则
+
+考虑到用户会有自己特有的规则定义需求，所以 Rsdoctor 除了内部已有的规则外，我们还提供了外部接口供用户定制自己的规则检查。
+外部扩展接口需要通过 `extends` 字段配置到 Rsdoctor 插件上，它的配置也是放在 `rules` 字段内。
+
+使用详情请查看：[自定义拓展规则](../../guide/custom/rule-custom.mdx)
```
Comment (chenjiahan) [inline packages/document/docs/zh/blog/release/release-note-0.3.mdx:8]: Absolute paths are easier to maintain. For example, you do not need to change the path after moving files.

```suggestion
使用详情请查看：[自定义拓展规则](/guide/custom/rule-custom)
```

File: packages/document/docs/en/blog/release/_meta.json [hunk 2]
```diff
@@ -1 +1 @@
-["release-note-1"]
+["release-note-0.1", "release-note-0.3"]
```
Comment (chenjiahan) [inline packages/document/docs/en/blog/release/_meta.json:1]: It is not recommended to include dot in URL, we can replace dot with `_`

Comment (easy1090) [reply packages/document/docs/en/blog/release/_meta.json]: Fixed

File: packages/document/docs/en/blog/release/release-note-0.3.mdx [hunk 3]
```diff
@@ -0,0 +1,130 @@
+# Rsdoctor v0.2-0.3 Release Note
```
[the remark this answers is not in the document]
Comment (easy1090) [reply packages/document/docs/en/blog/release/release-note-0.3.mdx]: Fixed

File: packages/document/docs/zh/blog/release/release-note-0.3.mdx (hunk 1 above)
Comment (easy1090) [reply packages/document/docs/zh/blog/release/release-note-0.3.mdx]: Fixed

**Review by chenjiahan — APPROVED** [review]
👍

**Pull request merged**